search-service/src/main/kotlin/com/egm/stellio/search/service/SubscriptionEventListenerService.kt

```diff
@@ -154,15 +154,16 @@ class SubscriptionEventListenerService(
             val payload = buildExpandedProperty(entitiesIds)
                 .addSubAttribute(NGSILD_OBSERVED_AT_PROPERTY, buildNonReifiedDateTime(notification.notifiedAt))
                 .first()
-            val attributeInstance = AttributeInstance(
-                temporalEntityAttribute = tea.id,
-                timeProperty = AttributeInstance.TemporalProperty.OBSERVED_AT,
-                time = notification.notifiedAt,
-                value = entitiesIds,
-                instanceId = notification.id,
-                payload = payload
-            )
-            attributeInstanceService.create(attributeInstance).bind()
+            attributeInstanceService.create(
+                AttributeInstance(
+                    temporalEntityAttribute = tea.id,
+                    timeProperty = AttributeInstance.TemporalProperty.OBSERVED_AT,
+                    time = notification.notifiedAt,
+                    value = entitiesIds,
+                    instanceId = notification.id,
+                    payload = payload
+                )
+            ).bind()
```

Comment:
you added a LOT of noise with all similar modifications in the PR. Whereas it seems nothing has changed...

Reply:
do you want me to remove this kind of change?

Reply:
unless there is a good reason for doing this move?

Reply:
I thought it was more readable that way.